opendatahub-io/odh-manifests
# JupyterHub: Add nvidia.com/gpu node toleration to gpu notebooks

FIX an issue where enabled gpus were not showing up in JH spawner UI

Adds the `nvidia.com/gpu EXISTS` node toleration to gpu notebooks in support of the support the node taint that is supported by the Nvidia gpu-operator

Verification Steps:
1. Deploy ODH w/ JupyterHub and cuda into a cluster with gpus enabled
2. When spawning a notebook confirm that the spawner Ui will allow you to select GPU the number of gpus
3. Confirm that the notebook pod has the toleration `nvidia.com/gpu`

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>
